docker/optimism/optimism.sh

```diff
@@ -95,15 +95,53 @@ e2e_test() {
     e2e_test_verify ${DEPLOYMENT_DIR} $l2_rpc $ADDRESS $e2e_test_request_id
 }
 
+
+e2e_test_fallback() {
+    set +eo pipefail
+    relayer=${ROOT}/relayer/relayer-node18-linux-x64
+    l2_rpc=http://localhost:9545
+    password=""
+
+    export SOURCE_CHAIN_ID=123
+    e2e_test_fill ${DEPLOYMENT_DIR} ${KEYFILE} "${password}" $l2_rpc
+    echo Sending Proof
+    
+    e2e_test_op_proof http://localhost:8545 $l2_rpc $PRIVKEY $e2e_test_l2_txhash
+    echo L1 Resolve
+    sleep 15
+    ${relayer} relay \
+        --l1-rpc-url http://localhost:8545 \
+        --l2-relay-to-rpc-url $l2_rpc \
+        --l2-relay-from-rpc-url $l2_rpc \
+        --wallet-private-key $PRIVKEY \
+        --l2-transaction-hash $e2e_test_l2_txhash 
+    set -eo pipefail
```

Comment:
I am assuming there is a reason for this, but would like to know it.

Reply:
pointed the reason in above comment